katello_certs_tools/katello_ssl_tool.py

```diff
@@ -267,8 +293,9 @@ def genPublicCaCert(password, d, verbosity=0, forceYN=0):
     fo.close()
 
     # permissions:
-    os.chmod(ca_cert, 0644)
-    os.chmod(latest_txt, 0644)
+    os.chmod(ca_cert, '0644')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)